drivers/bridge/bridge.go

```diff
@@ -405,12 +443,21 @@ func (d *driver) configure(option map[string]interface{}) error {
 			logrus.Warn(err)
 			return err
 		}
+		if config.EnableIPv6 {
```

Comment:
I wonder why this part has not been added to the existing setupIPForwarding function which does the same for IPv4.

Reply:
This seems to have been considered/solved in #2572.